### ISSUE TITLE
Multiple Prop Tables from a single <!-- PROPS --> call

### ISSUE THREAD

**Issue by joeycozza** [tuchk4/storybook-readme#159]
So we are using Storybook in a monorepo environment, and when I use "local" component Foo inside of Bar's stories, the  readme insertion in Bar's readme puts both Foo's and Bar's prop-types tables.

I was looking into the source code, and in services/getPropTables/index.js it appears that this is on purpose. At the same time, I do see that you started doing work with propTablesExclude, but in getDocsLayout.js it isn't passing in a config to the getPropsTables function.

Is this something you are actively working on/developing?

Would you appreciate a PR for this, or should I be patient and wait?

**Comment by tuchk4** [comment]
Correct. I would like to implement features like filterProps / filterComponent etc. for prop table.
Sadly, right now I don't have time to do this. I have plans to fix bugs and continue developing this addon after about two weeks.

If you have time and want to help with this PR I would be very happy :)

**Comment by joeycozza** [comment]
Okay, I'll see if I can get some of those options/features in.

**Comment by joeycozza** [comment]
Gotcha. I'm also going to be pretty busy/AFK for the next week or 2. If I get time to work on it, I'll give you a heads up, otherwise probably don't count on me doing it before you. Sorry.

**Comment by lonyele** [comment]
Maybe I can be a help here. While looking at the code, I think I want some opinions on how this feature should be. Should the feature be excluding first? or including first? or both?  

Think of how this feature will be used... what about this?
From global  `addParamters` (This is at `.storybook/config.js`)
```
excludedPropTables: ["Some", "React Component", "Name", "Here"]
```

From local `addParameters` (This is at per story)
```
includedPropTables: ["Bar", "Other Components"]
```

There can be some components that want it to be excluded from globally such as i18n, render props, Theming(?) etc...  Only exclude feature from global(no include feature here)

And from the per story, pass components that want it to be shown. Only include feature from local(no exclude feature here) Because usually per story is for a specific component, number of components that user wants to show is less than hiding which leads to less code to write(writing the list of components to be shown vs writing the list of components to be hidden) 

Also what about the naming?  hidePropTables, showPropTables? exclude, include?

**Comment by tuchk4** [comment]
- I think we should use `include` / `exclude` prefixes for naming.
- I think it should be possible to set config at global `addParameters` and per story. But I am not sure about merging global and per story config. Maybe some issues.

- `And from the per story, pass components that want it to be shown` you mean component constructors or just names?


Also would be nice to discuss the usage example. 


@lonyele could you please provide the code?  Also sent you invite to repo member  😸

**Comment by lonyele** [comment]
@tuchk4 I meant for just passing names. Come to think of it, passing actual component may be better because of less possibility for users making mistakes(React component is already imported and used at the story so just pass that to array vs manually write the name of that component(I feel like this increases typo))  

Since the weekend has passed, I'll try in one or two days. I think more discussions and your reviews can be done there.  😊

**Comment by tuchk4** [comment]
@lonyele this issue is resolved in #163 right?

**Comment by tuchk4** [comment]
fixed in `5.0.4`
